web/src/drafts.ts

```diff
@@ -89,16 +116,23 @@ export const draft_model = (function () {
             save(drafts);
         }
         return changed;
-    };
+    }
 
-    exports.deleteDraft = function (id) {
+    function deleteDraft(id: string): void {
         const drafts = get();
 
         delete drafts[id];
```

Comment:
`Do not delete dynamically computed property keys  @typescript-eslint/no-dynamic-delete`

Any recommendations for what to do here? I guess I could recreate the drafts object without that key and send that to `save`?

Reply:
I think the recommendation here is probably that we should rework these to store data in a `map`. Might be a nontrivial migration so I'd `eslint-ignore` and leave a comment and/or note on https://github.com/zulip/zulip/pull/28731/commits/a0948718e35e61095abf1969024bdc0bf9025d16#r1475096701.